providers/implementations/rands/seeding/rand_unix.c

```diff
@@ -321,6 +321,8 @@ static ssize_t sysctl_random(char *buf, size_t buflen)
 #     define __NR_getrandom    356
 #    elif defined(__aarch64__)
 #     define __NR_getrandom    278
+#    elif defined(__loongarch__) || defined(__loongarch64)
+#     define __NR_getrandom    278
```

Comment:
Why do you need this? Isn't the generic fallback below sufficient?

Reply:
> Why do you need this? Isn't the generic fallback below sufficient?

I saw there is a same define under __aarch64__ macro. just free to tell me if it is not accpeted.

Reply:
What about removing the aarch64 case and just add aarch64 and loongarch to the comment in the generic case?

Such as: `#    else /* generic (f.e. aarch64, loongarch, loongarch64) */

Reply:
done